code/modules/mob/living/carbon/human/species_types/ipc.dm

```diff
@@ -0,0 +1,40 @@
+/datum/species/ipc
+	// TOASTERS!
+	name = "IPC"
+	id = "ipc"
+	say_mod = "states"
+	heatmod = 2 // Went cheap with Aircooling
+	coldmod = 2 // Don't put your computer in the freezer.
+	burnmod = 1.5 // Overcoming augmented damage reduction - 1.5 puts it on par with unaugged organics.
+	brutemod = 1.5
+	species_traits = list(NOBREATH,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,EASYDISMEMBER,EASYLIMBATTACHMENT,NOPAIN,NO_BONES,NOCLONE,NOTRANSSTING,MUTCOLORS,TOXINLOVER) // NOHUNGER is temporary. A liver is needed to process chems (LIKE HOLY WATER)
+	mutant_bodyparts = list("ipc_screen", "ipc_antenna")
+	default_features = list("mcolor" = "#7D7D7D", "ipc_screen" = "Static", "ipc_antenna" = "None")
+	meat = null
```

Comment:
For the hilarity value I'd make cybersteaks, tbh.

Reply:
Actually, how about metal sheets?